VogtFactory/RentalCarFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class RentalCarFactory
+    {
+        public RentalCar CreateRentalCar(string type)
+        {
+            RentalCar rentalCar = null;
```

Comment:
Consider `short circuiting` the code.

```c#
            if (type.Equals("Toyota"))
            {
                return new Toyota();
            }
            if (type.Equals("Dodge"))
            {
                 return new Dodge(); 
            }
            return null;
```

Then, you dont need `RentalCar rentalCar = null;`

One reason to do that.. you wont go through all the extra lines of code if it is a toyota. You return immediately, "short circuiting" the rest of the code.